charter/index.html

```diff
@@ -392,7 +404,9 @@ <h2>Success Criteria</h2>
     <p>To promote interoperability, the group will document testing policies, processes, and procedures, taking <a href="https://solidproject.org/ED/qa">Solid QA</a> as an input.</p>
 
     <!-- Horizontal review -->
-    <p>Each specification should contain sections detailing all known security and privacy implications for implementers, web authors, and end users.</p>
+    <p>Each specification should contain sections detailing all known security and privacy implications for implementers, web authors, and end users.
+      Furthermore, the Working Group must produce a general Threat Model for ecosystems of Linked Web Storage applications.
```

Comment:
I don't follow "ecosystems". The plural also doesn't particular imply or require any interoperability.

Did you mean "various categories of LWS applications"?

Did you mean "the LWS application ecosystem"?

Did you mean "the LWS ecosystem"?

Something else?

Considering the deliverable is about LW*Storage*, wouldn't it me more meaningful to put emphasis on storage (and/or servers) as opposed to application? Or did you mean "software" or "classes of products" instead of "application"?

Reply:
Simone's suggestion was a threat model "at the ecosystem level", so I reused the term "ecosystem". I agree that the plural is problematic and should be removed.

While I believe that there is a rough consensus on what "the Solid ecosystem" is, "the LWS ecosystem" felt a bit weird, as LWS does not exist yet. Adding "applications" seemed to make it more concrete. But...

> Considering the deliverable is about LWStorage, wouldn't it me more meaningful to put emphasis on storage (and/or servers) as opposed to application? Or did you mean "software" or "classes of products" instead of "application"?

... that's a good point. "the ecosystem of LWS classes of product" feels painfully verbose.

Would "the ecosystem of LWS software" be ok?

Reply:
The IETF broadly uses the term application, as the spec is applied in some software. In solid, the term application has a much narrower meaning - an app. I think *the ecosystem of LWS software* sounds ok, we could also consider *the ecosystem of LWS implementations*.